### PR TITLE
Migrate rds_cluster* modules and tests

### PR DESCRIPTION
**Pull request by alinabuzachis**
Depends-On: https://github.com/ansible/zuul-config/pull/443

Migrate rds_cluster* modules and tests

### CODE REVIEW

**Review by gravesm — APPROVED** [review]

File: tests/integration/targets/rds_cluster_snapshot/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,481 @@
+---
+- module_defaults:
+    group/aws:
+      region: "{{ aws_region }}"
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+  collections:
+    - community.aws
```
Comment (abikouo) [inline tests/integration/targets/rds_cluster_snapshot/tasks/main.yml:9]: it seems that this dependency can be removed, as all modules involved in this test are part of the `amazon.aws` collection

File: tests/integration/targets/rds_cluster/roles/rds_cluster/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+- name: rds_cluster integration tests
+  module_defaults:
+    group/aws:
+      region: '{{ aws_region }}'
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
+      security_token: '{{ security_token | default(omit) }}'
+
+  collections:
+  - community.aws
```
Comment (abikouo) [inline tests/integration/targets/rds_cluster/roles/rds_cluster/tasks/main.yml:10]: it seems that this dependency can be removed, as all modules involved in this test are part of the `amazon.aws` collection, same as next test

**Review by abikouo — APPROVED** [review]

File: tests/integration/targets/rds_cluster/roles/rds_cluster/tasks/main.yml (hunk 2 above)
Comment (alinabuzachis) [reply tests/integration/targets/rds_cluster/roles/rds_cluster/tasks/main.yml]: Removed, thanks!

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Pull request merged**